README.md

```diff
@@ -6,20 +6,19 @@ This is a Gradle plug-in that provides convenience methods for adding JxBrowser
 
 ```kotlin
 plugins {
-    id("com.teamdev.jxbrowser") version "0.0.3"
+    id("com.teamdev.jxbrowser") version "0.0.4"
 }
 
 jxbrowser {
     // The JxBrowser version. A mandatory field.
-    version = "7.24"
+    version = "7.35.1"
 
     // The location of JxBrowser repository to use. It's either the US or Europe.
     // By default, it's the US.
-    repositoryLocation = Repository.US
+    repository = Repository.US
```

Comment:
Let's make the location use geography terms instead of political terms. Under geography I mean continents instead of countries. This makes more sense in terms or proximity and speed of download.

So, it should be `Repository.NORTH_AMERICA` and `Repository.EUROPE`, instead of `Repository.US`.